src/OrganisationRegistry.Api/Backoffice/Admin/Configuration/ConfigurationCommandController.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace OrganisationRegistry.Api.Backoffice.Admin.Configuration;
```

Comment:
I believe we came to the conclusion this morning, not to split controllers when we don't actually send commands.